util/envDynawoAlgorithms.sh

```diff
@@ -599,6 +599,7 @@ create_distrib() {
   cp ../bin/* bin/
   cp ../lib/* lib/.
   cp -r ../share/* share/
+  cp $DYNAWO_HOME/sbin/timeline_filter/timelineFilter.py bin/.
```

Comment:
Why timeline filter is not called after simulation ended in development environment as in execution environment ?

Reply:
It has to be called in the platform to ease readability of file, as the development environment is for developers they can call directly the script if needed. If we see it is required we can add it later